src/interfaces/request.ts

```diff
@@ -37,6 +37,13 @@ export type RequestTypes =
   | 'mongo'
   | 'postgres'
 
+// The success/failure result of a probe
+export enum ProbeRequestResult {
```

Comment:
Any problem if it changed to positive integer? Something like 0 (failed), 1 (success), 2 (unknown).

Reply:
No problems here. I've updated the PR